tutorials/2d/2d_sprite_animation.rst

```diff
@@ -79,11 +79,37 @@ released.
 
     extends KinematicBody2D
 
+    onready var _animated_sprite = $AnimatedSprite
+
     func _process(delta):
         if Input.is_action_pressed("ui_right"):
-            $AnimatedSprite.play("run")
+            _animated_sprite.play("run")
         else:
-            $AnimatedSprite.stop()
+            _animated_sprite.stop()
+
+ .. code-tab:: csharp
+
+    class Character : KinematicBody2D
+    {
+        private AnimatedSprite _animatedSprite;
```

Comment:
Do we use a `_` prefix for private variables in C#?

Reply:
The style guide says so: https://docs.godotengine.org/en/stable/getting_started/scripting/c_sharp/c_sharp_style_guide.html#naming-conventions